### PR TITLE
Beta updates

### PR DESCRIPTION
**Pull request by rschwabco**

### CODE REVIEW

File: docs/aserto-console/manage-organizations.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+sidebar_label: Managing Organizations
+---
+
+# Managing Organizations
+
+:::caution note
+When first signing up to Aserto, you'll be provisioned with a personal account, which doesn't allow inviting other members. During the free Beta period, you can open a single organization and invite team members into it.
```
Comment (ogazitt) [inline docs/aserto-console/manage-organizations.mdx]: you can create organizations and invite...

File: docs/aserto-console/manage-organizations.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+:::caution note:::
+
+To invite team members to use Aserto within the same organization, you will have to create an organization (if you haven't already). To do so, open on the Organization drop down and click "Create Organization".
```
Comment (ogazitt) [inline docs/aserto-console/manage-organizations.mdx]: Click the Organization dropdown and select "Create Organization".

File: docs/aserto-console/manage-organizations.mdx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+<img src="/create-organization.png" width="200px" />
+
+Type in your desired organization name and click "Lock it in!". From
```
Comment (ogazitt) [inline docs/aserto-console/manage-organizations.mdx]: remove "from"

File: docs/aserto-console/manage-organizations.mdx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+To manage the currently selected organization, choose the **Manage organization** menu item from the dropdown.
+
+## Inviting a user to an organization
+
+To invite a new user to an organization, go to **Manage Organization** screen, click the **Invite member...** button, and enter their email.
+
+<img src="/invite-member-form.png" width="500px" />
+
+If the user exists in the Aserto directory, they will be invited to join the organization as a Viewer. If they don't yet have an Aserto account, they will be invited to create one and change their password.
```
Comment (ogazitt) [inline docs/aserto-console/manage-organizations.mdx]: join the organization in the role that you select for them (Viewer, Editor, Admin, Owner).

File: docs/aserto-console/policies.mdx [hunk 5]
```diff
@@ -4,44 +4,125 @@ sidebar_label: Managing Policies
...
 
-The underpinning of Aserto's authorization model is a [policy](/docs/overview/policy). 
+The underpinning of Aserto's authorization model is the [policy](/docs/overview/policy).
 
-Policies are authored, stored, and versioned as code in a `git` repository.
+**Policies** are authored, stored, and versioned as code in a `git` repository.
+
+A **policy instance** is an instance of the decision engine's runtime associated with a tagged version of a policy (or a tag mask). You can create multiple policy instances, each associated with a different version of a policy.
+
+To create a policy instance, you must first create a policy repository. Once the policy repository is set up, you can create a policy instance and associate it with a tag (or a tag mask) in the policy repository.
```
Comment (ogazitt) [inline docs/aserto-console/policies.mdx]: tag masks are gone. when you bind to a tag, you're implicitly binding to the semver meaning of that tag.

Binding to "1" will bind to the latest semver version of the "1" lineage - e.g. "1.17.23".  Binding to "1.0" will bind to "1.0.27" if that 's the latest tag for that semver. Binding to "2.3.4" will bind directly to that tag.  

Binding to latest will implicitly bind to the latest tag (eg 3.4.5).

Comment (ogazitt) [reply docs/aserto-console/policies.mdx]: (this isn't a verbatim recommendation - just explaining how the tags work now)

File: docs/aserto-console/policies.mdx [hunk 6]
```diff
@@ -4,44 +4,125 @@ sidebar_label: Managing Policies
...
 
-Name your new repository with the prefix `policy-` (for example, `policy-peoplefinder`). This will 
-clone the template to the organization and repository name that you selected. The repository will 
-have the right GitHub Action installed, so that when you tag a commit, a new policy image will 
-automatically be built and pushed to the Aserto Registry. See the [Policy Lifecycle](/docs/overview/policy-lifecycle)
-topic for more information.
+<img src="/remove-policy-repository.png" />
 
-Finally, name your policy with a descriptive name (e.g. peoplefinder). 
+Note that removing a policy from your Aserto organization doesn't remove the policy repository. To remove the repository from your git provider, use that provider's API or UI.
```
Comment (ogazitt) [inline docs/aserto-console/policies.mdx]: doesn't remove the source code repository

File: docs/getting-started-acmecorp/create-account.mdx [hunk 7]
```diff
@@ -4,13 +4,11 @@ sidebar_label: Create an account
 
 # Create an Aserto account and tenant
 
-:::caution Note
-Aserto is in private beta. To complete the process of creating an account, you must have an invitation link.
-:::
+Go to [aserto.com/signup](https://aserto.com/signup) and complete the Sign Up experience.
```
Comment (ogazitt) [inline docs/getting-started-acmecorp/create-account.mdx]: I think this is the wrong URL.

File: docs/getting-started-acmecorp/create-policy.mdx [hunk 8]
```diff
@@ -2,32 +2,39 @@
...
+[<img src="/connect-scc.png" width="600px" />](/connect-scc.png)
+
+This will bring up a modal for adding a connection to a provider. Note that Aserto supports GitHub as a source code provider, but allows you to connect to it either over an OAuth2 flow, or using a Personal Access Token (PAT).
+
+<img src="/connect-github.png" width="600px" />
+
+The easiest path is to select "github" as the source code provider, give the connection a name (like `github-<youraccount>`), and click "Add connection". To connect using a Personal Access Token (PAT), follow these [instructions](/docs/aserto-console/connections#connecting-github-using-a-pat).
 
 :::note
 If you are managing an Aserto tenant for an organization, we recommend using a GitHub "bot account" to sign in with,
```
Comment (ogazitt) [inline docs/getting-started-acmecorp/create-policy.mdx:40]: If you are managing an Aserto organization, we recommend using ...

File: docs/getting-started-acmecorp/explore-abac-policies/update-app-and-review.mdx [hunk 9]
```diff
@@ -14,10 +14,12 @@ Then, scroll down to the Environment variables section and click edit variables:
 
 ![netlify-environment](/getting-started-acmecorp/netlify-environment.png)
 
-Update the `POLICY_ID` variable to the Policy ID of the policy you just created and save your changes. To find the policy ID, open your policy settings tab in the Aserto console:
+To retrieve the new Policy ID, open the Policies tab in the Aserto console and click the `peoplefinder-abac` instance. Then, click on `Policy Settings`:
 
 ![policy-settings-abac](/getting-started-acmecorp/policy-settings-abac.png)
 
+Update the `POLICY_ID` enviornment variable in Netlify to the Policy ID of the policy associated with the `peoplefinder-abac` instnace.
```
Comment (ogazitt) [inline docs/getting-started-acmecorp/explore-abac-policies/update-app-and-review.mdx]: instnace -> instance

**Review by ogazitt — CHANGES_REQUESTED** [review]

File: docs/aserto-console/manage-organizations.mdx (hunk 1 above)
Comment (rschwabco) [reply docs/aserto-console/manage-organizations.mdx]: Shouldn't the limit be mentioned?

File: docs/getting-started-acmecorp/create-account.mdx (hunk 7 above)
Comment (rschwabco) [reply docs/getting-started-acmecorp/create-account.mdx]: What's the correct URL?... This is the only one I ever used.

File: docs/aserto-console/manage-organizations.mdx (hunk 1 above)
Comment (ogazitt) [reply docs/aserto-console/manage-organizations.mdx]: the limit is 5. I don't think we need to mention it here.

File: docs/getting-started-acmecorp/explore-abac-policies/creating-an-abac-policy.mdx [hunk 10]
```diff
@@ -8,7 +8,7 @@ We're going to create an ABAC policy that will interact with our PeopleFinder ap
 
 ## Create the policy repository
 
-First, open the Aserto Console and navigate to the **Policies** tab. Click on the "Creat a new repository" button:
+First, open the Aserto Console and navigate to the Policies tab. Click on the "Creat a new repository" button:
```
Comment (ogazitt) [inline docs/getting-started-acmecorp/explore-abac-policies/creating-an-abac-policy.mdx:11]: Creat -> Create

File: docs/aserto-console/connections.mdx [hunk 11]
```diff
@@ -20,7 +20,7 @@ The Console currently supports two connection types - Identity Providers and Sou
 Aserto needs to connect to one or more identity providers as the source of user attributes and roles.
 
 :::note
-Currently Aserto supports Auth0 as an identity provider. More are on the way!
+Currently, Aserto supports Auth0 as an identity provider. More are on the way!
```
Comment (ogazitt) [inline docs/aserto-console/connections.mdx:23]: Auth0 and Okta

**Review by ogazitt — APPROVED** [review]

**Pull request merged**